src/functions/slack_function_handler_type_test.ts

```diff
@@ -190,3 +190,51 @@ Deno.test("SlackFunctionHandler with only outputs", () => {
   const result = handler(createContext({ inputs: {} }));
   assertEquals(result.outputs?.out, "test");
 });
+
+Deno.test("SlackFunctionHandler with only completed false", () => {
+  const TestFn = DefineFunction({
+    callback_id: "test",
+    title: "test fn",
+    source_file: "test.ts",
+    output_parameters: {
+      properties: {
+        completed: {
+          type: "boolean",
+        },
+      },
+      required: [],
+    },
+  });
+  const handler: SlackFunctionHandler<typeof TestFn.definition> = () => {
+    return {
+      completed: false,
+    };
+  };
+  const { createContext } = SlackFunctionTester(TestFn);
+  const result = handler(createContext({ inputs: {} }));
+  assertEquals(result.completed, false);
+});
+
+Deno.test("SlackFunctionHandler with only error", () => {
+  const TestFn = DefineFunction({
+    callback_id: "test",
+    title: "test fn",
+    source_file: "test.ts",
+    output_parameters: {
+      properties: {
+        error: {
+          type: "string",
+        },
+      },
+      required: [],
+    },
```

Comment:
nit: Same comment as above where we don't want the function to have a valid output named the same thing we're testing for.